datafusion/functions/src/lib.rs

```diff
@@ -90,14 +90,25 @@ pub mod macros;
 pub mod core;
 make_stub_package!(core, "core_expressions");
 
-make_package!(
-    encoding,
-    "encoding_expressions",
-    "Hex and binary `encode` and `decode` functions."
-);
+/// Encoding expressions.
```

Comment:
This is the actual change of this PR. The rest of the PR is the result of running `cargo fmt` and checking it in